packages/ui/src/TextField.tsx

```diff
@@ -314,6 +314,9 @@ export function TextField({
                     } else {
                       onChange({value: formattedPhoneNumber});
                     }
+                  } else if (type === "number") {
+                    text.replace(/[^0-9]/g, "");
```

Comment:
I think you need to do text = text.replace here?

Reply:
oo yes i think so. good catch